app/src/main/java/com/android/sample/ui/map/Map.kt

```diff
@@ -131,4 +155,119 @@ fun MapScreen(
             tabList = LIST_TOP_LEVEL_DESTINATION,
             selectedItem = Route.MAP)
       })
+
+  if (showBottomSheet) {
+    ModalBottomSheet(
+        onDismissRequest = { showBottomSheet = false },
+    ) {
+      Column(modifier = Modifier.fillMaxWidth().padding(16.dp)) {
```

Comment:
We have created constants that are used for padding please take a look at the C file in resources

Reply:
Thank you for this remark